default.nix

```diff
@@ -13,8 +13,6 @@ let
   inherit (reflex-platform) hackGet nixpkgs;
   pkgs = nixpkgs;
 
-  inherit (import dep/gitignore.nix { inherit (nixpkgs) lib; }) gitignoreSource;
-
```

Comment:
dead code

Reply:
If this isn't needed, do we still need the thunk? Is it imported elsewhere?

Reply:
Yes, it is imported elsewhere (can't find in this diff, but I saw it via grepping for `gitignoreSource`)

Reply:
I put it back for now.